client/ios/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -0,0 +1,121 @@
+{
+  "images" : [
```

Comment:
Those images are not shown (at least for beta build)

Reply:
because beta was created not from the latest state :)